hpx/runtime/applier/apply_helper.hpp

```diff
@@ -152,7 +152,7 @@ namespace hpx { namespace applier { namespace detail
         {
             // Direct actions should be able to be executed from a non-HPX thread
             // as well
-            if (this_thread::has_sufficient_stack_space() ||
+            if (//this_thread::has_sufficient_stack_space() ||
```

Comment:
We should enable that in the PR and fix it afterwards, but let's move on here...